pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -231,6 +232,13 @@ func (opt *Options) Run(args []string) error {
 	includeSuccess := opt.IncludeSuccessOutput
 	if len(tests) == 1 {
 		includeSuccess = true
+	} else {
```

Comment:
This doesn't belong here, it belongs as an `[Early]` test (which is what `[sig-arch][Early] Managed cluster should start all core operators [Suite:openshift/conformance/parallel]` does).

By putting this here you break jobs that are expected to have unsettled operators at the start, of which we don't *want* to wait for them.

Please move into the Early test

Reply:
If you had a concern about the early not running in a given suite, that's easy to fix.

Reply:
Also... you know this information about operators going unready is already in the event log for all runs, right?  The one you linked has multiple lines about the insights operator:

```
Jan 20 20:59:11.561 E clusteroperator/insights changed Degraded to True: PeriodicGatherFailed: Source clusterconfig could not be retrieved: Get "https://10.0.135.71:10250/containerLogs/openshift-sdn/sdn-64h84/sdn?limitBytes=65536&sinceSeconds=86400": dial tcp 10.0.135.71:10250: connect: connection refused
Jan 20 21:15:41.056 W clusteroperator/insights changed Degraded to False
```

So just add a standard synthetic (operators should not go degraded or unavailable during a run)